frb_codegen/src/ir/field.rs

```diff
@@ -64,4 +72,15 @@ impl IrField {
 
         format!("{enum_name}.{variant_name}")
     }
+
+    /// Takes the provided name and surrounds it with the `mirror_` prefix if the field is in a
+    /// mirrored enum.
```

Comment:
```suggestion
    /// Takes the provided name and surrounds it with the `mirror_` prefix if the field is in a
    /// mirrored enum.
    /// This may be improved in the future: https://github.com/fzyzcjy/flutter_rust_bridge/pull/1144#issuecomment-1486569869
```